rosdep/python.yaml

```diff
@@ -689,6 +689,18 @@ python-evdev:
     vivid: [python-evdev]
     wily: [python-evdev]
     xenial: [python-evdev]
+python-expiringdict:
+  debian:
+    jessie: [python-expiringdict]
+    sid: [python-expiringdict]
+    stretch: [python-expiringdict]
+  fedora:
+    pip:
+      packages: [expiringdict]
+  ubuntu:
+    xenial: [python-expiringdict]
+    yakkety: [python-expiringdict]
```

Comment:
I though it was a way to explicitly mention that this package was available for this versions only, isn't it ?